.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
```

Comment:
Can we review a similar repo list CAD and it's configuration and have OAO repo as dependency? Eg - https://github.com/openshift/configuration-anomaly-detection/blob/main/.github/dependabot.yml#L6

Reply:
Config file updated to include the OAO go module check